drracket-tool-doc/scribblings/drracket-tools/drracket-tools.scrbl

```diff
@@ -128,15 +128,36 @@ in order to make the results be platform independent.
     a @racket[_source-obj] argument.
   }
   
+ @defmethod[(syncheck:add-text-type [source-obj (not/c #f)]
+                                    [start exact-nonnegative-integer?]
+                                    [end exact-nonnegative-integer?]
+                                    [text-type (or/c 'matching-identifiers
+                                                     'unused-identifier
+                                                     'document-identifier)])
+            void?]{
+   Called to indicate that the color associated with the text type @racket[text-type]
+   should be drawn on the background of the given range in the editor,
+   when the mouse moves over it.
+
+   This method is usually called by Check Syntax to add background colors to an
+   identifier based on its lexical information. The types @racket['matching-identifiers],
+   @racket['unused-identifier] and @racket['document-identifier] correspond to
+   the color @racket['drracket:syncheck:matching-identifiers],
+   @racket['drracket:syncheck:unused-identifier] and @racket['drracket:syncheck:document-identifier]
+   in the preference, respectively.
+
```

Comment:
Is there a way to add a link to some relevant documentation for the "in the preference" phrase here?

Reply:
Sounds good. I added a link to <https://docs.racket-lang.org/drracket/color-scheme.html>.